dialpad/client.py

```diff
@@ -53,9 +53,10 @@ def _cursor_iterator(self, response_json, path, method, data, headers):
       data['cursor'] = response_json['cursor']
       response = self._raw_request(path, method, data, headers)
       response.raise_for_status()
-      response_json = response.json()
-      for i in response_json['items']:
-        yield i
+      if response != []:
```

Comment:
`if response` should suffice as an empty list is considered `False`.

Reply:
A much neater way to handle this would be 
```python
response_json = response.json()
for i in response_json.get('items', []):
  yield i
```
This way, we don't need an additional `if` conditional check.

Reply:
I interpreted the PR to mean that the `.json()` call is erroring on an empty response, but yes this would be good if it's the key access.

Reply:
Yeah my bad, in case the `.json()` returns a `NoneType`, we may just need to have an `or dict()` in the `response_json` assignment. 

P.S: edited the code snippet above accordingly.